### DISCUSSION TITLE
Daisy ui toast notification

### DISCUSSION THREAD

**Discussion by mariojgt** [saadeghi/daisyui#988] [Show and tell]
Hi devs just sharing one project of mine that is basically a vanilia js toast notification with almos 0 configuration, made with daisy ui.

https://mariojgt.github.io/wind-notify/

**Comment by saadeghi** [comment]
Awesome 👏

**Comment by uiuxphil** [comment]
Hey! Is there any version of this one or similar project that supports typescript?

**Comment by xydv** [comment]
This is awesome!

**Comment by Chippd** [comment]
Really like this. 
The mobile view could be improved a tad perhaps, it's a bit large no?
Also one thing I'd love to see (and would be willing to contribute to) is a way for multiple notifications of the same type to be stacked or to have a number increment in them. 
For example my app has image uploads where the user can upload 20+ images in one go. I have a toast for every image that is successfully uploaded or errors. My problem of course is multiple toasts spamming the user's screen. 

Now that I type this out I'm realising there are a few options
1. "stack" notifications that are the "same"
2. A way for updating an existing notification (so incrementing the count of images uploaded in my case)
3. Something else? 

Nice work though and thanks for sharing

**Comment by cd4success** [comment]
Amazing package! Thanks a lot.
